runner/templates/configmap.yaml

```diff
@@ -22,12 +22,42 @@ data:
       name = "{{ template "fullname" . }}"
       url = {{ .Values.gitlabUrl | quote }}
       token = {{ .Values.token | quote }}
-      executor = {{ default "docker" .Values.executor.type | quote}}
+      executor = {{ default "docker" .Values.executor.type | quote }}
```

Comment:
If you have executor `docker` and `values.executor.kubernetes` uncommented it will lead to broken state.

Maybe have a configuration like this and check for enabled state explicitly.
```
executor:
  docker:
    enabled: true
    ....
  kubernetes:
    enabled: true     
    ....
```